src/App/Fossa/Main.hs

```diff
@@ -27,38 +28,47 @@ appMain = do
 
   maybeApiKey <- checkAPIKey optAPIKey
 
+  let override = OverrideProject { overrideName = optProjectName, overrideRevision = optProjectRevision, overrideBranch = Nothing }
```

Comment:
I don't like this being on one line.  But I was getting syntax errors trying to move it around. 🤷

Reply:
you can make it multi-line if you indent the `{` (or whatever else you're breaking on) past the `o` in `override`:
```hs
let override = OverrideProject
      { overrideName = optProjectName
      , overrideRevision = optProjectRevision
      , overrideBranch = Nothing
      }
```

and then running ormolu, haha